src/client.js

```diff
@@ -270,25 +271,43 @@ Client.prototype.input = function(data) {
 	var client = this;
 	var text = data.text.trim();
 	var target = client.find(data.target);
+
 	if (text.charAt(0) !== "/") {
-		text = "/say " + text;
+		text = "say " + text;
+	} else {
+		text = text.substr(1);
+
+		// If after removing first / we still have a /, send it as a message
+		if (text.charAt(0) === "/") {
+			text = "say " + text;
+		}
 	}
+
 	var args = text.split(" ");
-	var cmd = args.shift().replace("/", "").toLowerCase();
+	var cmd = args.shift().toLowerCase();
+	var result;
+
 	_.each(inputs, function(plugin) {
 		try {
 			var path = "./plugins/inputs/" + plugin;
 			var fn = require(path);
-			fn.apply(client, [
+			result = fn.apply(client, [
 				target.network,
 				target.chan,
 				cmd,
 				args
 			]);
+			// Commands return true on successful execution
+			// lodash expects `false` to stop the loop
+			return !result;
 		} catch (e) {
 			console.log(path + ": " + e);
```

Comment:
If you consider my comment above, this `catch` should be moved along with the `require`s outside of the loop execution, making the `var result = inputs.some(...)` even more direct.
By doing so, you also make sure that any bug coming from a discrepancy between the input list and the available files in the plugins folder is detected as soon as The Lounge is started, and not whenever someone uses the buggy command.
Also, the current code silently absorbs the error without re-throwing it. All plugins are (at least at the moment) coming from us, so having the app crash miserably is acceptable here: there is no point in providing a recovery state from there IMO.

The current code is a bit worrying I think, because it fairly difficult to detect a silenced bug as is, unless you keep an eye on the log (which, by the way, is not even the error log...).